app/services/campaign_drafts/campaigns/launch.rb

```diff
@@ -27,7 +27,7 @@ def interview_form
         when 'single' then
           template = InterviewForm.find @campaign_draft.default_template_id
           new_form = InterviewForm.create(
-            template.attributes.except('id', 'created_at', 'updated_at').merge(used: true)
+            template.attributes.except('id', 'created_at', 'updated_at').merge(used: true, categories: template.categories)
```

Comment:
@harold93 where(used: true) should be a scope so you could call .used.where...
please create it :)

Reply:
but its not a where its a merge
btw I don't know its purpose

Reply:
Right! All good then :)